config/core.json

```diff
@@ -4,7 +4,7 @@
 		"analytics": false,
 		"dashboard": false,
 		"devdocs": false,
-		"onboarding": false,
+		"onboarding": true,
```

Comment:
I'll set this to false before merge. Setting to `true` for an example of at least one feature being active.